lib/mandrill_dm/message.rb

```diff
@@ -64,6 +64,17 @@ def html
       mail.html_part ? mail.html_part.body.decoded : mail.body.decoded
     end
 
+    def template
```

Comment:
Ouch. Ok, let's go with `template`. Please add an extra explanation to README that it's not `template_name` since it's used by ActionMailer.